css/amp.css

```diff
@@ -691,9 +691,9 @@ i-amphtml-video-mask, i-amp-video-mask {
   transition: background-color 1s;
 }
 
-.i-amphtml-dockable-video > video.i-amphtml-dockable-video-minimizing,
-.i-amphtml-dockable-video > iframe.i-amphtml-dockable-video-minimizing {
-  position: fixed;
+.i-amphtml-dockable-video-minimizing > video,
```

Comment:
(full review coming soon, jumping on some comments right now).

Yeah this is fairly important to figure out before we can merge this. Ideally we can have a way for a service to import CSS and inject into page, which may require some build changes.

Short term, we can move these into a single CSS file and have all components that implement the video-interface import it.